examples/layouts/sidebar_app.R

```diff
@@ -0,0 +1,80 @@
+library(shiny)
+library(shiny.semantic)
+library(glue)
+
+ui <- function() {
+  shinyUI(semanticPage(
+    title = "Sidebar Layout Test",
+    theme = "spacelab",
+    sidebar_layout(
+      sidebar_panel(
```

Comment:
:warning: This example doesn't work. Also we shouldn't use grid like that - it is very hard to understand what is happening here (especially that `sidebar_panel` arguments are passed with ellipsis (`...`) and then to `panel(...)`. At first read I thought some magic happens here :)

The way I imagine it working is to treat `grid()` function like other HTML elements:

```
sidebar_layout(
  sidebar_panel(
    "Hello world",
    sliderInput("obs", "Number of observations:", min = 0, max = 1000, value = 500)
    # Every argument here generates new div (row) in the sidebar, no need for grid here
  ),
  main_panel(
    h3(textOutput("caption")),
    grid(template,
           area1 = [code],
           area2 = [code],
           area3 = [code],
           area4 = [code]
     ),
    h5(textOutput("bottomCaption")) # Grid can be mixed with other HTML elements
  )
```